lib/matplotlib/tests/test_colorbar.py

```diff
@@ -712,6 +712,18 @@ def test_colorbar_label():
     assert cbar3.ax.get_xlabel() == 'horizontal cbar'
 
 
+@image_comparison(['colorbar_keeping_xlabel.png'], style='mpl20')
+def test_keeping_xlabel():
+    # github issue #23398 - xlabels being ignored in colorbar axis
+    rng = np.random.default_rng(19680801)
+    arr = rng.random((8, 8))
+    fig, axes = plt.subplots()
+    im = axes.imshow(arr)
```

Comment:
Generally use `ax` for a single Axes.
```suggestion
    fig, ax = plt.subplots()
    im = ax.imshow(arr)
```